grails-app/conf/DataSource.groovy

```diff
@@ -4,7 +4,7 @@ dataSource {
     driverClassName = "com.mysql.jdbc.Driver"
     dialect = org.hibernate.dialect.MySQL5InnoDBDialect
     username = "root"
-    password = "password"
+    password = "piku1995"
```

Comment:
We already started using local config for system specific passwords. @apoorvsrivastava1116 @PraveenTahalyani @akshitagangrade